template/package.json

```diff
@@ -3,30 +3,30 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "react-native start",
-    "ios": "react-native run-ios",
-    "android": "react-native run-android",
+    "start": "npx react-native start",
+    "ios": "npx react-native run-ios",
+    "android": "npx react-native run-android",
```

Comment:
This is totally unecessary. If you use yarn it doesn't make sense to do this. Same with npm.

Reply:
Ok. I will look at this again. I need to understand why I have issues when i try to use it.

Reply:
Once the install script finishes running it tells you use "npx react-native run-ios". This change would be consistent with the other install scripts suggestion.

At any rate, seems to work now as is so I am closing. Sorry about the distraction.

Reply:
It tells you so because it assume you don't have a "scripts" section yet, defined with the proper command.
npx is just a proxy that go into you node_modules/.bin if I am correct & that fallback to installing a tool from npm if no command available. It's totally unecessary in any package.json "scripts" section.